express/scripts/scripts.js

```diff
@@ -928,6 +928,13 @@ export function decorateButtons(block = document) {
   const noButtonBlocks = ['template-list', 'icon-list'];
   block.querySelectorAll(':scope a').forEach(($a) => {
     const originalHref = $a.href;
+    if ($a.children.length > 0) {
+      // We can use this to eliminate styling so only text
+      // propagates to buttons.
+      $a.querySelectorAll(':scope u').forEach(() => {
+        $a.innerHTML = $a.innerHTML.replace('<u>', '').replace('</u>', '');
+      });
```

Comment:
```suggestion
      $a.innerHTML = $a.innerHTML.replaceAll('<u>', '').replace('</u>', '');
```
No need for a loop here, you can just use `replaceAll()`.

Reply:
Oh duh, thanks @rofe